src/main/webapp/WEB-INF/views/notFoundPage.jsp

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Hello World JSP</title>
+</head>
+<body>
+<center>
+    <h1>Page not found :-( </h1>
+</center>
+</body>
+</html>
```

Comment:
Always add new line at the end of the file

Reply:
fixed